src/main/java/com/iexec/worker/tee/post/compute/worflow/FlowService.java

```diff
@@ -32,7 +32,7 @@ public ComputedFile readComputedFile(String taskId) throws PostComputeException
         ComputedFile computedFile = IexecFileHelper.readComputedFile(taskId, FileHelper.SLASH_IEXEC_OUT);
         if (computedFile == null) {
             final String cause = "computed.json missing";
-            log.error("ReadComputedFile failed (" + cause + ")");
+            log.error("ReadComputedFile failed ({})", cause);
```

Comment:
Do we still need this manual error log?

Reply:
I thought: "Yes, we do as `ReadComputedFile failed` is never printed otherwise`. But `POST_COMPUTE_COMPUTED_FILE_NOT_FOUND` will be printed, which should be sufficient. Let me clean those log lines!

Reply:
Please see https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/48/commits/56af36dd2b9374badb164ff5983226f241bb3517.